src/Native/Runtime/EHHelpers.cpp

```diff
@@ -443,6 +443,43 @@ Int32 __stdcall RhpVectoredExceptionHandler(PEXCEPTION_POINTERS pExPtrs)
     return EXCEPTION_CONTINUE_SEARCH;
 }
 
+#ifdef PLATFORM_UNIX
+Int32 __stdcall RhpHardwareExceptionHandler(UIntNative faultCode, UIntNative faultAddress, PAL_LIMITED_CONTEXT* palContext, UIntNative* arg0Reg, UIntNative* arg1Reg)
+{
+    CONTEXT contextRecord;
+    EXCEPTION_RECORD exceptionRecord;
+    EXCEPTION_POINTERS exceptionPointers;
+
+    exceptionPointers.ContextRecord = &contextRecord;
```

Comment:
I wonder whether it would be better to duplicate `RhpVectoredExceptionHandler` logic here instead of carefully setting up everything that it may need in Windows-specific structures.

(Ideally, we would eventually get to a place where `CONTEXT`, `EXCEPTION_RECORD` and `EXCEPTION_POINTERS` are not defined at all in Unix builds.)